packages/solecs/v2/tables/Vec2Table.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { console } from "forge-std/console.sol";
+import { IStore } from "../IStore.sol";
+import { StoreSwitch } from "../StoreSwitch.sol";
+import { StoreCore } from "../StoreCore.sol";
+import { SchemaType } from "../Types.sol";
+import { Bytes } from "../Bytes.sol";
+
+// -- User defined schema and id --
+
+bytes32 constant id = keccak256("mud.store.table.vec2");
+
+struct Schema {
```

Comment:
would this need to be called `Schema` to be picked up by autogen or could this be `Vec2`? thinking that we'll probably be passing these kinds of data around and having everything named `Schema` would be annoying

Reply:
Agree, it should a named struct for each table instead of `Schema` everywhere!

Reply:
For autogen I was imagining a sol-type-file/ts/json which is the config with all the schemas, from which all the .sol files are spawned